blocks/columns/columns.css

```diff
@@ -36,6 +36,13 @@ div.columns-container:has(.slim-promo) {
   margin-left: auto;
 }
 
+/** for FireFox (lack of) support for :has selector **/
```

Comment:
I would remove the :has selector above, to only use `.black-bg` selector. This guarantees a consistent look across browsers. 

Then also scope `.black-bg` to only `.columns`.